fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -56899,6 +56901,7 @@ name: adult anterior midgut epithelium
 def: "Columnar epithelium with a clear apical border (i.e. no brush border)." [FlyBase:FBrf0007735]
```

Comment:
Incomplete def. Consider changing to:  "The epithelium of the adult anterior midgut.  This is a columnar epithelium with..."